tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -16,6 +16,19 @@ function goHomeJoinNewGame() {
   cy.log('Joined new game successfully');
 }
 
+function validateMatch(match, length, p1, p2) {
+  expect(match.player1.id).to.eq(p1);
+  expect(match.player2.id).to.eq(p2);
+  expect(match.startTime).to.be.greaterThan(0);
+  expect(match.endTime).to.eq(null);
+  expect(match.games.length).to.eq(length);
+}
+
+function validateWin({status, winner}, gameWinner) {
```

Comment:
So this validates that a game has the expected status and winner?
```suggestion
function validateGameResult({status, winner}, expectedWinner) {
```